build.msbuild

```diff
@@ -1,17 +1,38 @@
-<Project>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <ItemGroup>
         <VersionFile Include="version.props"/>
     </ItemGroup>
     <PropertyGroup>
         <NuspecPath>Microsoft.jQuery.Unobtrusive.Ajax.nuspec</NuspecPath>
+        <DistDir>$(MSBuildThisFileDirectory)dist\</DistDir>
+        <ArtifactsDir>$(MSBuildThisFileDirectory)artifacts\build\</ArtifactsDir>
+        <WebPackTestDir>$(MSBuildThisFileDirectory)test\webpacktest\</WebPackTestDir>
     </PropertyGroup>
+
+    <ItemGroup>
+        <DistDirFiles Include="$(DistDir)*.*;"/>
+    </ItemGroup>
+
     <Target Name="Build">
         <ReadLinesFromFile File="@(VersionFile)">
             <Output TaskParameter="Lines" PropertyName="PackageVersion"/>
         </ReadLinesFromFile>
         <Exec Command="npm install" />
         <Exec Command="npm version --no-git-tag-version --allow-same-version $(PackageVersion)" />
-        <Exec Command="npx gulp" />
-        <Exec Command="nuget pack $(NuspecPath) -Version $(PackageVersion)" />
+        <Exec Command="$(MSBuildThisFileDirectory)node_modules\.bin\gulp" />
```

Comment:
The more cross-plat-friendly way to do this is `npm run build` and then put the call to `gulp` in the `build` task in your `package.json`. This takes care of knowing where to find the `gulp` executable - you don't have to hardcode the `node_modules\.bin\gulp` path.

Reply:
Neat!